.tool-versions

```diff
@@ -1,3 +1,3 @@
 erlang 26.2
 elixir 1.15.7-otp-26
-nodejs 14.17.3
\ No newline at end of file
+nodejs 18.20.1
```

Comment:
Why Node 18 in particular? Is this the minimum version for Webpack 5?

Reply:
Webpack is ok but the latest versions of our plugins support a minimum of 18. My first reaction was to bring down the plugin versions, but I saw that node 16 is actually not [maintained anymore](https://endoflife.date/nodejs). Even 18 is in maintenance mode, but we have a year until we are EOL for that version.

Reply:
I see. I had some issues on `screens` trying to update Node higher than 16, but I think they were related to Webpack, which I didn't want to tackle at that time just to get the repo working on Apple Silicon.